RELEASE.md

```diff
@@ -38,11 +41,16 @@ Ensure that the build you are releasing is passing the tests on the evergreen wa
 Ensure you have the `master` branch checked out, and that you have pulled the latest commit from `mongodb/mongo-odbc-driver`.
 
 #### Create the tag and push
+
+Git tags are used to specify which version of [`libmongosqltranslate`](https://github.com/10gen/mongosql-rs) to use.
+
```

Comment:
Do you think this is necessary since each section shows the format?

Reply:
It might not be necessary if you change the git tag snippet to show the format precisely (more or less) instead of providing an example.
As long as it is visible that the format matters, I think we are good.